### PR TITLE
Created works and about sections

### PR DESCRIPTION
**Pull request by emirsagit**
### Created works and about section

- Created Works section
- Styled works section container using CSS Grid
- Styled works section cards using Flexbox
- Created About section
- Styled about section using Flexbox
- Fixed Linters error

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -58,6 +60,171 @@ <h1 class="header-title">
...
+            </div>
+        </div>
+    </section>
+
+    <section class="about">
+        <h2 class="about-title">About <br> Myself</h2>
+        <p class="about-p">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+
+        <div class="social about-social">
+            <a class="social-title" href="#">LET’S CONNECT</a>
```
Comment (ElijahTobs) [inline index.html]: - [x] Since `hx` tags are for titles, texts like `LET'S CONNECT` should use a `hx` tag

File: index.html [hunk 2]
```diff
@@ -58,6 +60,171 @@ <h1 class="header-title">
...
+
+        <div class="about-tabs-wrapper">
+            <div class="tabs-title-wrapper tabs-title-first-wrapper">
+                <a class="tabs-title" href="#">Langguages</a>
+                <button class="tabs-button" type="button">
+                    <svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+                        <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M19 9l-7 7-7-7" />
+                    </svg>
+                </button>
+            </div>
```
Comment (ElijahTobs) [inline index.html]: - [x] The technology categories ("Languages", "Frameworks" and "Skills") is a list and should use the `ul` for grouping, while each of them should have a `li` tag.
- [x] The technology in each category (e.g., "JavaScript", "HTML" and "CSS") should also be embedded inside the `Languages`

**Review by ElijahTobs — CHANGES_REQUESTED** [review]
# Status: Changes Required ⚠️ 🚦
## Hi @emirsagit  👋

You've done really well so far! 👍

Your README is professional and you've followed the Github flow rules. ✔️ 
Great Pull Request (PR) title and summary ✔️ 

_However, there are some changes that you are required to make before this project can be approved._

## Required Changes ♻️

_See the inline comments below_

## Optional Changes 👌
_Every comment with the OPTIONAL prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend that you take them into account as they can make your code better._

- Your commit messages should be in the imperative tense and not past tense. e.g:
  - Add about myself section

#### If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below

#### "Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."

### Note that you still have 2 more reviews as written in the [Code Reviews Limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561#), however, if you feel this review is unfair, you can request a second opinion using this [form](https://airtable.com/shrQAqnBwek5a0O0s).

## Happy Coding :computer: 

_Reviewed by_  **Elijah O. Ayandokun**

File: index.html (hunk 1 above)
Comment (descholar-ceo) [reply index.html]: :+1: This issue has been addressed successfully :heavy_check_mark:

File: index.html (hunk 2 above)
Comment (descholar-ceo) [reply index.html]: :+1: This issue has been addressed successfully :heavy_check_mark:

File: index.html [hunk 3]
```diff
@@ -1,63 +1,415 @@
...
+						sign-ups required.
+					</p>
+					<div class="work-tags">
+						<a class="work-tag" href="#">html</a>
+						<a class="work-tag" href="#">css</a>
+						<a class="work-tag" href="#">javascript</a>
+					</div>
+					<button type="button" class="bg-white-button">See Project</button>
+				</div>
+			</div>
```
Comment (descholar-ceo) [inline index.html]: - [ ] All of the work should be in a ul tag and li for each project

**Review by descholar-ceo — APPROVED** [review]
Hi @emirsagit , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

=> You have implemented all of the changes requested in the previous review, but I noticed just one issue which was not spotted during the first review, we apologise about that, but you still need to fix it, before you merge all the changes to the default branch., kindly check it out in my inline comment, below!

=> And also I noticed that all of your commit messages are in past tense, that is not recommended, please learn more about professional commit message using [this article](https://www.freecodecamp.org/news/writing-good-commit-messages-a-practical-guide/), and after going through it, you are supposed to edit your messages and make sure that they use present tense or imperative mode. To edit commit message, you can checkout [this cool video](https://www.loom.com/share/00771f369b7b47fc968c019ecf0d3ba6) Explaining how you can edit your commit messages

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (emirsagit) [reply index.html]: Thank you very much. I changed it as you said. :100:

**Pull request merged**